root/example/index.html

```diff
@@ -1,10 +1,39 @@
 <!doctype html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <title>{%= name %}</title>
+<meta charset="UTF-8">
+<title>{%= name %}</title>
+<style>
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  .scribe {
+    min-height: 60px;
+    border: 1px solid #ececec;
+  }
+</style>
 </head>
-<body>
 
+<body>
+<h1>{%= name %}</h1>
+<div class='scribe'></div>
+<script src='../bower_components/requirejs/require.js'></script>
+<script>
+  require.config({
+    baseUrl: '../',
+    paths: {
+        'scribe': 'bower_components/scribe/scribe',
```

Comment:
This needs to be changed as the `bower.json` requires `scribe: 1.0.0` not `scribe: master`